css/sticky-footer.css

```diff
@@ -0,0 +1,30 @@
+/* Sticky footer styles
+-------------------------------------------------- */
+html {
+  position: relative;
+  min-height: 100%;
+}
+body {
+  /* Margin bottom by footer height */
+  margin-bottom: 60px;
+}
+.footer {
+  width: 100%;
+  /* Set the fixed height of the footer here */
+  height: 60px;
+  line-height: 60px; /* Vertically center the text there */
+  background-color: #f5f5f5;
+  padding: 0;
+  margin: 0;
+}
+
+
+/* Custom page CSS
+-------------------------------------------------- */
+/* Not required for template or sticky footer method. */
+
+.container {
+  width: 100%;
+  /*max-width: 680px;*/
+  padding: 0 6%;
+}
```

Comment:
Bootstrap's sticky footer is very specific to the sample they provide. For your case, you don't need both of these styles. You should just stick to giving your footer a reasonable padding and margin and making sure it has `width: 100%`. Outside of that, none of this in necessary.